CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ And please only add new entries to the top of this list, right below the `# Unre
 
 # Unreleased
 
+- On Windows, apply skip taskbar state when taskbar is restarted.
```

Comment:
could you remove the changelog entry here? `set_skip_taskbar` will make its first appearance in the upcoming release

Reply:
okay